components/layout_thread_2020/lib.rs

```diff
@@ -874,7 +872,12 @@ impl LayoutThread {
             display_list.build_stacking_context_tree(&fragment_tree, &self.debug);
 
         // Build the rest of the display list which inclues all of the WebRender primitives.
-        let is_contentful = display_list.build(context, &fragment_tree, &root_stacking_context);
+        let is_contentful = display_list.build(
+            context,
+            &fragment_tree,
+            &root_stacking_context,
+            highlighted_dom_node,
```

Comment:
Instead of passing this down into the display list builder, could we add a `FragmentFlag` that indicates that a fragment should be highlighted? I think this will work a bit better if we ever needed to highlight more than one node?

Reply:
I thought about this, but figured checking the flag for every fragment might have a nonzero performance impact? TBH I can't think of a reason why we would want to highlight more than one node.

Reply:
Actually, the longer I think about it the more sense it makes to do this with a flag on a fragment

Reply:
No, that doesn't work. One node can create multiple fragments, so highlighting them independently would look wrong.